src/client/browser/idle-page/index.js

```diff
@@ -1,24 +1,26 @@
 import * as browser from '../';
+import StatusMarkupCreator from './status-markup-creator';
 import COMMAND from '../../../browser-connection/command';
 
 
-const CHECK_STATUS_INTERVAL = 1000;
+const CHECK_STATUS_DELAY = 1000;
 
-
-var interval = null;
+var statusMarkupCreator = null;
 
 function checkStatus (statusUrl) {
     browser
         .checkStatus(statusUrl, window.XMLHttpRequest)
         .then((cmd) => {
-            if (cmd !== COMMAND.idle)
-                window.clearInterval(interval);
+            if (cmd === COMMAND.idle)
+                window.setTimeout(() => checkStatus(statusUrl), CHECK_STATUS_DELAY);
+        })
+        .catch(()=> {
+            statusMarkupCreator.changeStatus(false);
```

Comment:
Invalid arrow style. Please, rebase to pick #108 changes, so you will have lingting for this.